lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_current_branch_is_hotfix.rb

```diff
@@ -7,7 +7,8 @@ module SharedValues
     class AndroidCurrentBranchIsHotfixAction < Action
       def self.run(params)
         require_relative '../../helper/android/android_version_helper'
-        Fastlane::Helper::Android::VersionHelper.is_hotfix?(Fastlane::Helper::Android::VersionHelper.get_release_version)
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
+        Fastlane::Helper::Android::VersionHelper.is_hotfix?(Fastlane::Helper::Android::VersionHelper.get_release_version(app))
```

Comment:
💄 nitpick, given that the test about `ENV['PROJECT_NAME']` is not necessary after all, this below might end up being a nicer split of the code statements:
```suggestion
        version = Fastlane::Helper::Android::VersionHelper.get_release_version(params[:app])
        Fastlane::Helper::Android::VersionHelper.is_hotfix?(version)
```